maven-projects/spark/datasources-32/src/main/scala/org/apache/spark/sql/graphar/GarTable.scala

```diff
@@ -31,8 +31,12 @@ import org.apache.spark.sql.execution.datasources.v2.FileTable
 import org.apache.spark.sql.graphar.csv.CSVWriteBuilder
 import org.apache.spark.sql.graphar.orc.OrcWriteBuilder
 import org.apache.spark.sql.graphar.parquet.ParquetWriteBuilder
+import org.apache.spark.sql.graphar.json.JSONWriteBuilder
 import org.apache.spark.sql.types._
 import org.apache.spark.sql.util.CaseInsensitiveStringMap
+import org.apache.spark.sql.execution.datasources.json.JsonDataSource
+//import org.apache.spark.sql.execution.datasources.v2.json.JsonWrite
```

Comment:
Is the JsonWrite no need to import? If so, just remove it.

Reply:
done